KerbalStuff/blueprints/api.py

```diff
@@ -410,6 +411,30 @@ def user_info_api(username):
     return info
 
 
+@api.route('/api/user/<username>/change-password', methods=['POST'])
+@with_session
+@user_required
+@json_output
+def change_password(username):
+    if current_user.username != username:
+        return {'error': True, 'reason': 'You are not authorized to change this user\'s password.'}, 403
+
+    old_password = request.form.get('old-password')
+    password1 = request.form.get('new-password')
+    password2 = request.form.get('new-password-confirm')
+
+    if not bcrypt.hashpw(old_password.encode('utf-8'), current_user.password.encode('utf-8')) == current_user.password.encode('utf-8'):
+        return {'error': True, 'reason': 'The old password you entered doesn\'t match your current account password.'}
+
+    pw_valid, pw_message = check_password_criteria(password1, password2)
+    if pw_valid is True:
```

Comment:
```suggestion
    if pw_valid:
```

Do we need `is True`?

Reply:
Not really.

Reply:
Yeah I think I included it to make sure that whatever we get back really is `True` and not something else like a string or something, but I realize it's unnecessary.